pkgs/tools/misc/scfbuild/default.nix

```diff
@@ -2,23 +2,17 @@
 
 buildPythonApplication {
   pname = "scfbuild";
-  version = "1.0.3";
+  version = "2.0.0";
 
   format = "other";
 
   src = fetchFromGitHub {
     owner = "13rac1";
     repo = "scfbuild";
-    rev = "9acc7fc5fedbf48683d8932dd5bd7583bf922bae";
-    sha256 = "1zlqsxkpg7zvmhdjgbqwwc9qgac2b8amzq8c5kwyh5cv95zcp6qn";
+    rev = "6d84339512a892972185d894704efa67dd82e87a";
+    sha256 = "0wkyzkhshlax9rvdmn441gv87n9abfr0qqmgs8bkg9kbcjb4bhad";
   };
 
-  patches = [
-    # Convert to Python 3
-    # https://github.com/13rac1/scfbuild/pull/19
-    ./python-3.patch
```

Comment:
Please delete the patch.

Reply:
Done! (`fixup! scfbuild: 1.0.3 -> 2.0.0`)